examples/maven-spring-compile/src/main/java/community/flock/wirespec/example/maven/custom/app/common/WirespecSerializer.java

```diff
@@ -6,32 +6,166 @@
 import community.flock.wirespec.java.Wirespec;
 import org.springframework.stereotype.Component;
 
+import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 @Component
 public class WirespecSerializer implements Wirespec.Serialization<String> {
```

Comment:
It's a bit tedious to maintain these serializers (every example basically needs a copy). The idea is that you don't have to depend on a library to use Wirespec but with the serializers becoming more complex. I think the other side of the trade-off is more attractive. In integration-wirespec I've now created a `DefaultQueryParamSerde` class, if we decide to let go the 0-dependency stance these example serializers will be much easier to maintain.

Reply:
We've decided that most use cases benefit from the default serializer and don't need a custom implementation. Therefore the examples now depend on the default serializer from wirespec integration.